core/templates/base-components/base-content.component.css

```diff
@@ -106,3 +106,13 @@
     display: flex !important;
   }
 }
+
+@media screen and (min-width: 600px) and (max-width: 768px) {
+  .navbar-container .collapse.d-lg-inline-flex.navbar-collapse.oppia-navbar-collapse.ng-cloak.navbar-options-display nav-options {
```

Comment:
Why are we adding this styling here in the base component? We should not be doing that if the issue exists only in the topic editor